src/core/components/AppAddrQr.vue

```diff
@@ -71,6 +71,12 @@ import { mapState } from 'vuex';
 import { toChecksumAddress } from '@/core/helpers/addressUtils';
 
 export default {
+  props: {
+    currency: {
```

Comment:
you can get the currency name from 'global' store. You can fuzzy search `currencyName` to see examples.